mesheryctl/internal/cli/root/system/context/create.go

```diff
@@ -0,0 +1,44 @@
+package context
+
+import (
+	"github.com/layer5io/meshery/models"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	url       = ""
+	tokenPath = ""
+)
+
+// createContextCmd represents the update command
+var createContextCmd = &cobra.Command{
+	Use:   "create",
+	Short: "create context",
+	Long:  `create context and add to config`,
+	Args:  cobra.MaximumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		log.Printf("Adding Context %s to config", args[0])
+		log.Println("Setting as Current Context")
```

Comment:
We can skip the in-progress messages and simply return a final success (or error) message.

Reply:
Made the change